File: 3264. Final Array State After K Multiplication Operations I/Solution.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        for _ in range(k): 
            minVal = float("inf")
            minIdx = None
            for i in range(len(nums)): 
                if nums[i] < minVal: 
                    minVal = nums[i]
                    minIdx = i 
            if minIdx is not None:
                nums[minIdx] *= multiplier
        return nums

File: 3264. Final Array State After K Multiplication Operations I/Solution.cpp
class Solution {
public:
    vector<int> getFinalState(vector<int>& nums, int k, int multiplier) {
        for(int i=0;i<k;i++){
            int idx = -1, val = INT_MAX; 
            for(int j=0;j<nums.size();j++){
                if (nums[j] < val){
                    val = nums[j]; 
                    idx = j;
                }
            }
            nums[idx]*=multiplier; 
        }
        return nums; 
    }
};
